lib/middleware/preview.js

```diff
@@ -208,6 +218,12 @@ function previewMiddleware(config, req, res, next) {
  *   is used.
  * @param {Object} [config.helpers] - optional map of Handlebars helpers to
  *   register
+ * @param {Object} [config.globalModel] - optional model that will be extended
+ *   into the root object before rendering. If your ingredients contain @root
+ *   references, you'll want to specify those things here.
+ * @param {Object} [config.commonModel] - optional model that will be extended
```

Comment:
I don't think we need this. The ingredients should instead require a common base model in their `model.js` entrypoint.